frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -61,12 +47,12 @@ export const FormComponentConfig = ({
     optionsId,
     hasCustomFileEndings,
     validFileEndings,
+    children,
     ...rest
   } = schema.properties;
 
-  const unsupportedPropertyKeys: string[] = Object.keys(rest).filter((propertyKey) => {
-    return !isPropertyTypeSupported(rest[propertyKey]);
-  });
+  // children property is not supported in component config - it should be part of container config.
+  const unsupportedPropertyKeys: string[] = getUnsupportedPropertyTypes(rest, children ? ['children'] : undefined);
```

Comment:
Great work! 👏 Loved the readability and the comment that explains the children property! 🥇